cobbler.spec

```diff
@@ -404,6 +404,7 @@ fi
 %config(noreplace) %{_sysconfdir}/cobbler/boot_loader_conf/yaboot_ppc.template
 %config(noreplace) %{_sysconfdir}/cobbler/cheetah_macros
 %config(noreplace) %{_sysconfdir}/cobbler/dhcp.template
```

Comment:
Please rename to `dhcp_v4`.

Reply:
I learned that this is the same in the isc dhcp server. So let's leave it like this.